tests/js_handle_test.go

```diff
@@ -54,3 +54,56 @@ func TestJSHandleEvaluate(t *testing.T) {
 		})
 	}
 }
+
+func TestJSHandleEvaluateHandle(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name     string
+		pageFunc string
+		args     []any
+		expected string
+	}{
+		{
+			name: "no_args",
+			pageFunc: `handle => {
+				return {"innerText": handle.innerText};
+			}`,
+			args:     nil,
+			expected: `{"innerText":"Some title"}`,
+		},
+		{
+			name: "with_args",
+			pageFunc: `(handle, a, b) => {
+				return {"innerText": handle.innerText, "sum": a + b};
+			}`,
+			args:     []any{1, 2},
+			expected: `{"innerText":"Some title","sum":3}`,
+		},
+	}
+	for _, tt := range tests {
+		tt := tt
+
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+
+			tb := newTestBrowser(t)
+			p := tb.NewPage(nil)
+
+			err := p.SetContent(`<html><head><title>Some title</title></head></html>`, nil)
+			require.NoError(t, err)
+
+			result, err := p.EvaluateHandle(`() => document.head`)
+			require.NoError(t, err)
+			require.NotNil(t, result)
+
+			got, err := result.EvaluateHandle(tt.pageFunc, tt.args...)
+			assert.NoError(t, err)
```

Comment:
I'd make this `require`.

Reply:
Sure, updated in b5ff202df57e398bb4f40c7a0e875a6c547bdd54